pkg/issuer/acme/setup.go

```diff
@@ -363,10 +360,10 @@ func ensureEmailUpToDate(ctx context.Context, cl client.Interface, acc *acmeapi.
 // account with the clients private key already exists, it will attempt to look
 // up and verify the corresponding account, and will return that. If this fails
 // due to a not found error it will register a new account with the given key.
-func (a *Acme) registerAccount(ctx context.Context, cl client.Interface, eabAccount *acmeapi.ExternalAccountBinding) (*acmeapi.Account, error) {
+func (a *Acme) registerAccount(ctx context.Context, cl client.Interface, acmeEmail string, eabAccount *acmeapi.ExternalAccountBinding) (*acmeapi.Account, error) {
```

Comment:
since we do no longer have access to `a.issuer` here, we have to pass `acmeEmail string`.